lib/compiler_rt/fixdfdi.zig

```diff
@@ -8,6 +9,10 @@ comptime {
         @export(&__aeabi_d2lz, .{ .name = "__aeabi_d2lz", .linkage = common.linkage, .visibility = common.visibility });
     } else {
         @export(&__fixdfdi, .{ .name = "__fixdfdi", .linkage = common.linkage, .visibility = common.visibility });
+
+        if (builtin.cpu.arch.isArmOrThumb() and builtin.target.isMinGW()) {
```

Comment:
Just looking over the code, is `mingw` + `eabi` a valid combination? In LLVM they can be separately true:
https://github.com/llvm/llvm-project/blob/df935ff4eca91013553edbf3ca1d45b568f6709f/compiler-rt/lib/builtins/fixdfdi.c#L38

Reply:
In practice, no. We don't generally copy LLVM's export logic in compiler-rt 1:1 since it's overly broad in some cases.